### DISCUSSION TITLE
Frigate on NUC

### DISCUSSION THREAD

**Discussion by cwleth** [blakeblackshear/frigate#755] [Ask A Question]
Hi all,

Im looking to get my four Hikvision cameras running through Frigate with real time object detection, but struggle to find the best setup. Currently I’m running HA OS in a Proxmox VM on a nuc5 i3 8gb ram.

As being quite new to this world I believe that the HassOS Addon would be my best and easiest option - for this I have a few questions:

1. Documentation says “Running Frigate in a VM on top of Proxmox … is not recommended” should I dedicate the NUC and install HA OS image directly to get the Addon or is there a better way?
2. Documentation recommends using Google Coral - would that be needed on the nuc5 i3 with 8gb ram?
3. If yes - i presume it’s the USB accelerator? and is there a guide to set this up with HA somewhere?

Appreciate your help

**Comment by HIT2022** [comment]
1. Can't speak to downsides of Frigate in a Proxmox _LXC_ container, but I'm running HA Supervised (w/ the Frigate Addon) in a Debian VM on an i7 Nuc (Proxmox host) with no issues
2. Performance w/ or w/o Coral is a hard question to answer because it depends on # of cameras, resolution, fps, etc. But on an i3 Nuc - I would not expect Coral to be a necessity. In fact I'm not sure it a necessity for pis, and I think the idea is more to significantly cut down on _avoidable_ CPU/electricity use. Frigate w/ 4 1080p cameras uses up about 6% on my i7, but I'm not sure what that % would look like without Coral and/or GPU hardware acceleration
3. USB is the default path, but I believe they are difficult to acquire right now. If your Nuc has an M.2 slot, you can use the PCIe version instead
